src/plugins/hotkeyPlugin.ts

```diff
@@ -265,27 +219,57 @@ export class HotkeyManager {
   /**
    * エディタが変更されたときに呼び出される。
    */
-  onEditorChange(editor: "talk" | "song"): void {
-    this.hotkeys.setScope(editor);
+  onEditorChange(editor: EditorType): void {
+    // this.hotkeys.setScope(editor);
+    this.scope = editor;
     this.log("Editor changed to", editor);
   }
+
+  keyInput(e: KeyboardEvent): void {
+    const element = e.target;
+    // メニュー項目・ダイアログではショートカットキーを無効化
+    if (
+      element instanceof HTMLElement &&
+      (element.getAttribute("role") == "menu" ||
+        element.classList.contains("q-dialog__inner"))
+    ) {
+      return;
+    }
+
+    const isInTextbox =
+      element instanceof HTMLElement &&
+      (element.tagName === "INPUT" ||
+        element.tagName === "SELECT" ||
+        element.tagName === "TEXTAREA" ||
+        element.contentEditable === "true");
+
+    const combination = combinationToBindingKey(eventToCombination(e));
+
+    const actions = this.actions
+      .filter((item) => !isInTextbox || item.enableInTextbox)
+      .filter(
+        (item) =>
+          combinationToBindingKey(this.getSetting(item).combination) ==
+          combination,
+      )
+      .filter((item) => item.editor.split("&").includes(this.scope));
```

Comment:
`talk&song`というリテラル型に対して&が含まれているかという情報を使っているので、意図しないバグが混入しやすくなってるかなと！
例えばここに&判定があることを知らず、`talk&song`を`talk and song`に変えても気づかない感じです。

ちょっと泥臭いですがif elseで判定する形が無難かなと！
```suggestion
      .filter((item) => {
        if (item.editor === "talk&song") {
          return true;
        } else if (item.editor === "talk") {
          return this.scope === "talk";
        } else if (item.editor === "song") {
          return this.scope === "song";
        }
      });
```